admin/category_product_listing.php

```diff
@@ -175,9 +175,7 @@
 
             if ($do_delete_flag) {
               //--------------PRODUCT_TYPE_SPECIFIC_INSTRUCTIONS_GO__BELOW_HERE--------------------------------------------------------
-              if (file_exists(DIR_WS_MODULES . $zc_products->get_handler($product_type) . '/delete_product_confirm.php')) {
-                require DIR_WS_MODULES . $zc_products->get_handler($product_type) . '/delete_product_confirm.php';
-              }
+              require zen_get_admin_module_from_directory($product_type, 'delete_product_confirm.php');
```

Comment:
Not sure if this was a bug, but this never fired before, because (probably nobody) ever had an override copy of this file.

Or, does that mean what I've done here actually CAUSES a bug, because it WILL now fallback to the default when it was only intended to run IF there was a file specific to the product-type?

Reply:
@zcwilt Thoughts?

Reply:
I think I answered my own question: it was a bug, but completely obscured because of duplicated code.

I've removed the duplication in favor of having it fall back onto the module-specific override behavior.